ci-operator/step-registry/enable-qe-catalogsource/disconnected/enable-qe-catalogsource-disconnected-commands.sh

```diff
@@ -196,11 +196,12 @@ spec:
     registryPoll:
       interval: 15m
 EOF
+    set +e 
     COUNTER=0
-    while [ $COUNTER -lt 600 ]
+    while [ $COUNTER -lt 30 ]
     do
-        sleep 1
-        COUNTER=`expr $COUNTER + 1`
+        sleep 20
+        COUNTER=`expr $COUNTER + 20`
```

Comment:
In this logic, the total timeout is shorten to 30s? and extend interval to 20s, why here counters plus 1? Should not keep 1 here?

Reply:
Yes, it should be +1 here when the total value is `30`.